connection-service/new-admin-vm/new-admin-vm.json

```diff
@@ -72,11 +72,11 @@
                 "typeHandlerVersion": "2.9",
                 "autoUpgradeMinorVersion": true,
                 "settings": {
-                    "modulesUrl": "[concat(parameters('CAMDeploymentBlobSource'),'/Install-CAM-CS.ps1.zip')]",
+                    "modulesUrl": "[concat(parameters('CAMDeploymentBlobSource'),'/Install-ConnectionServer.ps1.zip')]",
```

Comment:
I changed the name (again) mostly to avoid having to create a new blob storage. But we do have to stick with a name in the end - what do you think of this one?

Reply:
I prefer to Install-ConnectionServer.ps1.zip